tests/modules/individuals/resources/test_merge.py

```diff
@@ -0,0 +1,270 @@
+        individual2 = Individual.query.get(individual2_id)
+        assert not individual2
+
+    finally:
+        individual_utils.delete_individual(flask_app_client, researcher_1, individual1_id)
+        # individual2 is gone now due to successful merge!
+        sighting1.delete_cascade()
+        sighting2.delete_cascade()
+
+
+@pytest.mark.skipif(
+    test_utils.module_unavailable('individuals', 'encounters', 'sightings'),
+    reason='Individuals module disabled',
+)
+def test_merge_permissions(
+    db, flask_app_client, researcher_1, researcher_2, contributor_1
+):
+    individual1_id = None
+    sighting1 = None
+    encounter1 = None
+    individual2_id = None
+    sighting2 = None
+    encounter2 = None
+    try:
+        sighting1, encounter1 = prep_data(db, flask_app_client, researcher_1)
+        individual1_id = str(encounter1.individual_guid)
+        sighting2, encounter2 = prep_data(db, flask_app_client, researcher_2)
+        individual2_id = str(encounter2.individual_guid)
+
+        # this tests as researcher_2, which should trigger a merge-request (owns just 1 encounter)
+        # NOTE: merge_request not yet implmented, so fails accordingly (code 500)
+        data_in = [individual2_id]
+        response = test_utils.post_via_flask(
+            flask_app_client,
+            researcher_2,
+            scopes=('individuals:write',),
+            path=f'/api/v1/individuals/{individual1_id}/merge',
+            data=data_in,
+            expected_status_code=500,
+            response_200={'merged'},
+        )
+        assert response.json['merge_request']
+        assert response.json['message'] == 'Merge failed'
+        assert response.json['blocking_encounters'] == [str(encounter1.guid)]
+
+        # a user who owns none (403 fail, no go)
+        response = test_utils.post_via_flask(
+            flask_app_client,
+            contributor_1,
+            scopes=('individuals:write',),
+            path=f'/api/v1/individuals/{individual1_id}/merge',
+            data=data_in,
+            expected_status_code=403,
+            response_200={'merged'},
+        )
+
+        # anonymous (401)
+        response = test_utils.post_via_flask(
+            flask_app_client,
+            None,
+            scopes=('individuals:write',),
+            path=f'/api/v1/individuals/{individual1_id}/merge',
+            data=data_in,
+            expected_status_code=401,
+            response_200={'merged'},
+        )
+
+    finally:
+        individual_utils.delete_individual(flask_app_client, researcher_1, individual1_id)
+        individual_utils.delete_individual(flask_app_client, researcher_2, individual2_id)
+        sighting1.delete_cascade()
+        sighting2.delete_cascade()
+
+
+@pytest.mark.skipif(
+    test_utils.module_unavailable(
+        'individuals', 'encounters', 'sightings', 'social_groups'
+    ),
+    reason='Individuals module disabled',
+)
+def test_merge_social_groups(db, flask_app_client, researcher_1):
```

Comment:
This is a models test not a resources test and should be in a different file

Reply:
moved to individuals/test_merge instead.